packages/vscode-docker-registries-extension/src/extension.ts

```diff
@@ -23,6 +23,7 @@ export function activate(context: vscode.ExtensionContext) {
 	context.subscriptions.push(vscode.commands.registerCommand('dockerRegistries2.refreshRegistries', () => urtdp.refresh()));
 	context.subscriptions.push(vscode.commands.registerCommand('dockerRegistries2.connectRegistry', () => urtdp.connectRegistryProvider()));
 	context.subscriptions.push(vscode.commands.registerCommand('dockerRegistries2.disconnectRegistry', (item) => urtdp.disconnectRegistryProvider(item)));
+	context.subscriptions.push(vscode.commands.registerCommand('dockerRegistries2.disconnectGenericV2Registry', (item) => urtdp.disconnectRegistryProvider(item)));
```

Comment:
The command shouldn't call the standard `disconnectRegistryProvider` method, since the provider itself isn't being disconnected from--just the individual registry. It should call into `removeTrackedRegistry`.

Reply:
yeah that makes sense lol